src/app/shared/editor/components/yaml-editor/config-editor.events.ts

```diff
@@ -0,0 +1,10 @@
+import * as monaco from "monaco-editor";
+export const YAML_EDITOR_OPTIONS: monaco.editor.IStandaloneEditorConstructionOptions = {
+    theme: "vs",
+    language: "yaml",
+    renderLineHighlight: "none",
+    minimap: {
+        enabled: false,
+    },
+    scrollBeyondLastLine: false,
+};
```

Comment:
Why the file is called "events"? It contains "options"

Reply:
I just replaced the file, it was already named like this, and with the same content.
But I will re-name it